hub/api/integrations.py

```diff
@@ -597,7 +597,8 @@ def _init_ds(self):
         if not self._inited:
             self._inited = True
             self._samples_in_chunks = {
-                key: value.chunks[0] for key, value in self._ds._tensors.items()
+                key: (None in value.shape) and 1 or value.chunks[0]
```

Comment:
Is this the only revert required? Have you tested speeds are applying these changes?

Reply:
Yes, after reverting this, the speed becomes the same as 1.3.0 on imagenet benchmarks